css/main.scss

```diff
@@ -7,7 +7,7 @@
 
 // Our variables
 $base-font-family: Helvetica, Arial, sans-serif;
-$base-font-size:   16px;
+$base-font-size:   16;
```

Comment:
yes but its a bug to set a variable with a px value because its not an int variabel anymroe and cannot be part of computations by sass (that's why i removed the px).
Instead we should add the px suffx when we use the variable.